data/tests_without_attestation.yaml

```diff
@@ -1,25 +1,19 @@
 ---
 default_input_args:
-  attestation: n
-  runtime_args: n
```

Comment:
How is the redis_default running without runtime_args and runtime_env variable? Don't we need them to pass Ctrl+G.

Reply:
Now, with latest curation app, these screen are not present. You can directly give these inputs

Reply:
Updated runtime_variables_list and runtime_args_text